blog-article.html

```diff
@@ -43,7 +43,7 @@ <h1 class="logo">
             </nav>
             <div class="tools">
                 <form class="search">
-                    <button type="submit" class="search__btn">
+                    <button type="submit" method="post" class="search__btn">
```

Comment:
The buttons send the form. And that's why we specify a method for the form, not for the buttons